File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/dto/SceneCaseDto.java
package org.javamaster.invocationlab.admin.dto;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

/**
 * 测试场景详细信息标识
 *
 * @author yudong
 */
@Data
public class SceneCaseDto {
    String caseName;
    List<UserCaseDto> caseDtoList = new ArrayList<>();
    String sceneScript;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/dto/SceneCaseDto.kt
package org.javamaster.invocationlab.admin.model.dto

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * 测试场景详细信息标识
 *
 * @author yudong
 */

@AllOpen
class SceneCaseDto {
    var caseName: String? = null
    var caseDtoList: List<UserCaseDto> = ArrayList()
    var sceneScript: String? = null
}
